### ISSUE TITLE
colspan not applying

### ISSUE THREAD

**Issue by romeJG** [danomatic/react-pdf-html#101]
here is my code and the col span does not apply below are my code and the output
the foo with a col span should have 2 col span
```jsx
  let html = `
        <html>
        <body>
        <style>
            table {
                border:2px solid #000; 
                border-collapse: collapse;
                width: 100%;
            }
            th{
                background: #004f9e;
                padding: 2px;
                color : white;
            }
            td {
                padding: 2px;
            }
        </style>
            <table>
                <thead>
                    <tr>
                        <th>Column 1</th>
                        <th>Column 2</th>
                        <th>Column 3</th>
                    </tr>
                </thead>
            <tbody>
                <tr>
                    <td>Foo</td>
                    <td>Bar</td>
                    <td>Foobar</td>
                </tr>
                <tr>
                    <td colspan="2">Foo</td>
                    <td>Bar</td>
                </tr>
                <tr>
                    <td>Some longer thing</td>
                    <td>Even more content than before!</td>
                    <td>Even more content than before!</td>
                </tr>
            </tbody>
            </table>
            <br/>
            <img src="${imageUrl}" />
        </body>
        </html>
    `;

    return (
        <Document>
            <Page>
                <Html>
                    {html}
                </Html>
            </Page>
        </Document>
    );
```
![image](https://github.com/user-attachments/assets/391df99c-5fc9-46bb-b014-7ed255c13dba)

**Comment by danomatic** [comment]
Hi @romeJG. This is the current implementation of colspan:

https://github.com/danomatic/react-pdf-html/blob/main/src/renderers.tsx#L105
```typescript
  if (element.attributes && element.attributes.colspan) {
    const colspan = parseInt(element.attributes.colspan, 10);
    if (!isNaN(colspan)) {
      overrides.flexBasis = colspan;
    }
  }
```

I'm going to try some alternate approaches.

Note: The table implementation has limitations because it has to work within the constraints of a `flex` layout and I'm not using any special libraries to try and calculate content/row/col widths.

**Comment by danomatic** [comment]
I just published a new version `2.1.0` that uses percentages for the initial width and colspan. Please give it a try and let me know how it works for you.

**Comment by romeJG** [comment]
Thanks! just updated it and it worked
![image](https://github.com/user-attachments/assets/d1ab3041-1494-4af3-9b11-1c8a9a27d47e)
